datasets/amazon_reviews_multi/README.md

```diff
@@ -0,0 +1,235 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+  all_languages:
+  - de
+  - en
+  - es
+  - fr
+  - ja
+  - zh
+  de:
+  - de
+  en:
+  - en
+  es:
+  - es
+  fr:
+  - fr
+  ja:
+  - ja
+  zh:
+  - zh
+licenses: []
```

Comment:
Maybe add an `other-amazon-conditions` license here ?
(it starts with `other` since it's not part of the licenses list we have)